cmd/restic/global.go

```diff
@@ -496,7 +496,7 @@ func OpenRepository(ctx context.Context, opts GlobalOptions) (*repository.Reposi
 		if errors.IsFatal(err) {
 			return nil, err
 		}
-		return nil, errors.Fatalf("%s", err)
+		return nil, errors.FatalWrap(err)
```

Comment:
This line here was previously "erasing" the `ErrNoKeyFound` type and making the new error just a fatalError (string) type.

I ama Go (and restic) novice, so if this is a bad approach to the issue, please suggest a new one and I'll implement it.

Reply:
I think a simple `fmt.Errof("Fatal: %w")` would have the same effect. You will be able to check for the if the error `ErrKeyNotFound` is part of the error tree using `errors.Is(err, ErrKeyNotFound)`

Reply:
I'd like to eventually get rid of the fatal error type. so let's just use `fmt.Errorf` instead here.

Reply:
> I think a simple fmt.Errof("Fatal: %w") would have the same effect. You will be able to check for the if the error ErrKeyNotFound is part of the error tree using errors.Is(err, ErrKeyNotFound)

Correct, but the result wouldn't be a `fatalError`, which this block was specifically trying to generate - it either returns the error if it's fatal or if it's not fatal, wraps it in a fatal error.

> I'd like to eventually get rid of the fatal error type. so let's just use fmt.Errorf instead here.

But if that's the case, yes, we can just use `Errorf` - I will retool this PR and drop the wrapping.